org.adempiere.base/src/org/compiere/process/ProcessInfo.java

```diff
@@ -691,6 +692,80 @@ public ProcessInfoLog[] getLogs()
 		return logs;
 	}	//	getLogs
 
+	
+	/**
+	 * Save Log to DB
+	 * @param Log_ID
+	 * @param P_ID
+	 * @param P_Date
+	 * @param P_Number
+	 * @param P_Msg
+	 * @param tableId
+	 * @param recordId
+	 */
+	public void saveLog (int Log_ID, int P_ID, Timestamp P_Date, BigDecimal P_Number, String P_Msg,int tableId,int recordId)
+	{
+		saveLog (new ProcessInfoLog (Log_ID, P_ID, P_Date, P_Number, P_Msg,tableId,recordId));
+	}
+	
+	/**
+	 * Save Log to DB
+	 * @param P_ID
+	 * @param P_Date
+	 * @param P_Number
+	 * @param P_Msg
+	 * @param tableId
+	 * @param recordId
+	 */
+	public void saveLog (int P_ID, Timestamp P_Date, BigDecimal P_Number, String P_Msg ,int tableId,int recordId)
+	{
+		saveLog (new ProcessInfoLog (P_ID, P_Date, P_Number, P_Msg,tableId, recordId));
+	}
+	
+	/**************************************************************************
+	 * 	Save Log to DB
+	 *	@param Log_ID Log ID
+	 *	@param P_ID Process ID
+	 *	@param P_Date Process Date
+	 *	@param P_Number Process Number
+	 *	@param P_Msg Process Message
+	 */
+	public void saveLog (int Log_ID, int P_ID, Timestamp P_Date, BigDecimal P_Number, String P_Msg)
+	{
+		saveLog (new ProcessInfoLog (Log_ID, P_ID, P_Date, P_Number, P_Msg));
+	}	//	saveLog
+
+	/**
+	 * 	Save Log to DB
+	 *	@param P_ID Process ID
+	 *	@param P_Date Process Date
+	 *	@param P_Number Process Number
+	 *	@param P_Msg Process Message
+	 */
+	public void saveLog (int P_ID, Timestamp P_Date, BigDecimal P_Number, String P_Msg)
+	{
+		saveLog (new ProcessInfoLog (P_ID, P_Date, P_Number, P_Msg));
+	}	//	saveLog
+
+	/**
+	 * 	Save Log to DB
+	 *	@param logEntry log entry
+	 */
+	public void saveLog (ProcessInfoLog logEntry)

```

Comment:
javadoc should highlight the difference with existing addLog method where this is write to db immediately.

Reply:
ok, fixed